roles/keystone/templates/etc/keystone/uwsgi/keystone-main.ini

```diff
@@ -13,7 +13,7 @@ logfile-chmod = 644
 {% if keystone.uwsgi.method == 'socket' %}
 socket = /run/uwsgi/keystone-main.socket
 {% else %}
-http-socket = 127.0.0.1:{{ keystone.uwsgi.http_port.public }}
+socket = 127.0.0.1:{{ keystone.uwsgi.http_port.public }}
```

Comment:
Does the uwsgi-socket directive work here? Might be good to lay down our ini files like they suggest upstream if possible (https://github.com/openstack/keystone/blob/master/httpd/keystone-uwsgi-admin.ini#L6)